examples/server-side-rendering/__tests__/ssr.spec.ts

```diff
@@ -0,0 +1,44 @@
+import fetch from 'node-fetch';
+import { JSDOM } from 'jsdom';
+
+describe('SSR', () => {
+  describe('with no state', () => {
+    let dom;
+    beforeAll(async () => {
+      const res = await fetch('http://localhost:4000/');
+      const html = await res.text();
+      dom = new JSDOM(html);
```

Comment:
could this have been done with https://yarnpkg.com/en/package/jest-environment-jsdom-thirteen and `document.innerHTML = ...`?

Reply:
same as responded previously, I don't think I need an actual browser context.
Is there a benefit I've missed into using a jsdom env?

Reply:
seems like code could be simpler in the test